package.json

```diff
@@ -1,24 +1,16 @@
 {
   "name": "@cypress/request",
   "description": "Cypress's fork of a simplified HTTP request client.",
-  "keywords": [
-    "http",
-    "simple",
-    "util",
-    "utility"
-  ],
   "version": "0.0.0-development",
-  "author": "Mikeal Rogers <mikeal.rogers@gmail.com>",
   "repository": {
     "type": "git",
     "url": "https://github.com/cypress-io/request.git"
   },
   "bugs": {
     "url": "http://github.com/cypress-io/request/issues"
   },
-  "license": "Apache-2.0",
   "engines": {
-    "node": ">= 6"
+    "node": ">= 18"
```

Comment:
This looks like a breaking change which would stop Cypress being installed in a Node.js 16 environment. It would need the version of @cypress/request bumping to `4.0.0`
- https://github.com/cypress-io/cypress/issues/29601 schedules that change for the next major version of Cypress.